required/latex-lab/latex-lab-firstaid.dtx

```diff
@@ -166,17 +167,27 @@
     \def\@begintheorem##1##2[##3]{%
 %    \end{macrocode}
 % We use the theorem instance.
+% \changes{v0.85h}{2024/10/16}{Added pre and post skips tagging/733}
 %    \begin{macrocode}
-      \UseInstance{blockenv}{theorem}{}
+      \UseInstance{blockenv}{theorem}{beginsep=\thm@preskip}
+%    \end{macrocode}
+% There is no working key to set the endskip, so we set the skip directly
```

Comment:
that should be a temp situation: the end-skip key should work. So wouldn't it be better to adjust that in the block code to make it work first?

Reply:
I looked at it but got a bit lost at the various (in part unused) skips you added there. I will open an proper issue that end-skip doesn't work and leave a note in the block code to adjust the firstaid once it works.